driver/arguments.py

```diff
@@ -81,19 +80,29 @@
       "--evaluator", '"hff=ff()"', "--search", '"eager_greedy([hff], preferred=[hff])"']),
 ]
 
-EPILOG = """component options:
+
+def _format_example(description, parameters):
+    call_string = " ".join([Path(sys.argv[0]).name] + parameters)
+    return f"{description}\n{call_string}"
+
+
+def _format_examples(examples):
+    return "\n\n".join(_format_example(*example) for example in examples)
+
+
+EPILOG = f"""component options:
   --translate-options OPTION1 OPTION2 ...
   --search-options OPTION1 OPTION2 ...
                         pass OPTION1 OPTION2 ... to specified planner component
                         (default: pass component options to search)
 
 Examples:
 
-%s
-""" % "\n\n".join("%s\n%s" % (desc, " ".join([os.path.basename(sys.argv[0])] + parameters)) for desc, parameters in EXAMPLES)
+{_format_examples(EXAMPLES)}
+"""
```

Comment:
Have you tested that the output of this before and after is the same? I find it difficult to validate details.

Reply:
Yes, there is no difference in the output. This change is not really related to the issue, I just didn't like the nested `join` calls. The actual change related to the issue is to replace `os.path.basename(sys.argv[0])` with `Path(sys.argv[0]).name`.